### PR TITLE
[docs] Move text-to-image LoRA training from blog to docs

### PR DESCRIPTION
**Pull request by stevhliu**
This PR moves the relevant code example for text-to-image training with LoRA from the blog to the docs. To get it to fit in with the existing docs on LoRA, I had to rearrange the structure a bit, so now it shows training and inference for both text-to-image and DreamBooth. Let me know if I've forgotten anything!

### CODE REVIEW

File: docs/source/en/training/lora.mdx [hunk 1]
```diff
@@ -10,169 +10,205 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+>>> model_base = "runwayml/stable-diffusion-v1-5"
 
-</Tip>
+>>> pipe = StableDiffusionPipeline.from_pretrained(model_base, torch_dtype=torch.float16)
+>>> pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
+```
 
-But there is no free lunch. For the given dataset and expected generation quality, you'd still need to experiment with
-different hyperparameters. Here are some important ones:
+Load the LoRA weights from your finetuned model *on top of the base model weights*, and then move the pipeline to a GPU for faster inference. When you merge the LoRA weights with the frozen pretrained model weights, you can optionally adjust how much of the weights to merge with the `scale` parameter:
```
Comment (stevhliu) [inline docs/source/en/training/lora.mdx:105]: The merging bit was a little confusing to me, so I want to make sure I understand this correctly! 

The `scale` parameter controls how much of the base model weights + fine-tuned LoRA weights to use.

* `0`, then it's just the base model weights
* 1`, then it's the fully fine-tuned model (base model weights + LoRA). 

So is the following code equivalent for using the fully fine-tuned LoRA model?

```python
>>> image = pipe(prompt, num_inference_steps=25, guidance_scale=7.5).images[0]
>>> image = pipe(prompt, num_inference_steps=25, guidance_scale=7.5, cross_attention_kwargs={"scale": 1.0}).images[0]

**Comment by patrickvonplaten** [comment]
Or does it work out of the box?

**Comment by patrickvonplaten** [comment]
@yiyixuxu please take a look here as wel

**Comment by stevhliu** [comment]
> Would you instead like to iterate on https://huggingface.co/docs/diffusers/training/lora ?

Hmm, not sure I understand. 😅 These changes are being applied to the `lora.mdx` docs.

> Nice, can we here again link to the automatically generated google colab?

Yes, I just added the auto-generated Colab link which should be live once merged!

**Comment by sayakpaul** [comment]
> Hmm, not sure I understand. 😅 These changes are being applied to the lora.mdx docs.

My bad, @stevhliu! Please disregard.

**Comment by yiyixuxu** [comment]
@patrickvonplaten yes it links itself

**Review by yiyixuxu — COMMENTED** [review]
Awesome! thank you @stevhliu 
@pcuenca should we update the code example with CPU offloading?

**Comment by yiyixuxu** [comment]
@stevhliu feel free to merge this one in whenever

**Pull request merged**